app/services/idv/vendor_result.rb

```diff
@@ -0,0 +1,26 @@
+module Idv
+  class VendorResult
+    attr_reader :success, :errors, :reasons, :session_id, :normalized_applicant
+
+    def self.new_from_json(json)
+      parsed = JSON.parse(json, symbolize_names: true)
+
+      applicant = parsed[:normalized_applicant]
+      parsed[:normalized_applicant] = Proofer::Applicant.new(applicant) if applicant
+
+      new(**parsed)
+    end
+
+    def initialize(success:, errors:, reasons:, session_id:, normalized_applicant:)
+      @success = success
+      @errors = errors
+      @reasons = reasons
+      @session_id = session_id
+      @normalized_applicant = normalized_applicant
+    end
+
+    def success?
+      success
+    end
+  end
+end
```

Comment:
Just to state my goals clearly, the end result that I want is an object that can be can easily be serialized and passed around (from the background process) so we can continue working on the results of the proofing process outside of a background job. In this case I settled on creating `Idv::VendorResult`.

If we chose to make this class accept `result` as you suggest, then `@result` would be its only instance variable, and it would not serialize cleanly. The `result` (which is an `Proofer::Resolution`) has `result.vendor_resp` which is kind of a free-form field and so we have no way of easily deserializing it (it has different classes and values depending on which method we call on the client).

I am satisfied with the choice here to ignore reek's warning.